recipe/repack.sh

```diff
@@ -21,4 +21,7 @@ fi
 if [[ "$PKG_NAME" == "dpcpp_impl_linux-64" ]]; then
   # Not sure why these libraries are there, but seems to be unused.
   rm -f $PREFIX/lib/libffi.so*
+  HOST=x86_64-conda-linux-gnu
+  echo "--gcc-toolchain=${PREFIX} --sysroot=${PREFIX}/${HOST}/sysroot -target ${HOST}" > ${PREFIX}/bin/$HOST-icpx.cfg
+  echo "--gcc-toolchain=${PREFIX} --sysroot=${PREFIX}/${HOST}/sysroot -target ${HOST}" > ${PREFIX}/bin/$HOST-icx.cfg
```

Comment:
I thin it should be `${BUILD_PREFIX}/bin/$HOST-icpx.cfg`.

Reply:
This is when the compiler itself is being built. PREFIX at compiler build time becomes BUILD_PREFIX at downstream package build time.